### DISCUSSION TITLE
Why use Github? 3+ reasons why you should use Github

### DISCUSSION THREAD

**Discussion by MilesWK** [community/community#109493] [Other Feature Feedback, Questions, & Ideas]
Github is a really cool place... but why should you make an account and use it?

1.  Making tutorials
Repositories are a great way to make tutorials on how to make programs! You can style it with markdown, at pictures, and also add files into the repository such as example code, and much more!

2. Share your code
Did you create a really cool project that you would like to share with the world? Create a repository or some other project and describe what your code does and how to use it!

3. Manage projects
Let's say you created a python project that allowed you to create menus with just one function! You share it onto Github but there is an ERROR in the code!!! A fellow Github user can go to your project, head to the issues tab, and let you know of the issue! There you can reply to them, and mark the issue as resolved!

You can also update your repository into new versions of the code! Perfect for Python packages and anything of the sort!!

There are many more reasons why you should use Github! Check out the replies below for more!

**Comment by its21yash** [comment]
GitHub offers a plethora of benefits for developers:
1. Version Control: GitHub provides robust version control tools, allowing you to track changes to your code, revert to previous versions if needed, and collaborate with others seamlessly.
2. Collaboration.
3. Project Management.
4. Showcasing Your Work.
5. Learning and Tutorials: GitHub hosts a vast array of open-source projects and repositories, making it an excellent resource for learning new technologies, best practices, and coding techniques. You can also contribute to existing projects to gain practical experience and improve your skills.
6. Community Engagement.

GitHub is a valuable platform for developers of all levels, offering tools and resources to enhance productivity, collaboration, and learning in the world of software development.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @MilesWK , we're happy you're here! You are more likely to get a useful response if you are posting your questions in the applicable category. I've gone ahead and moved it for you. Good luck!
